app/controllers/idv/by_mail/enter_code_controller.rb

```diff
@@ -144,7 +144,7 @@ def pii_locked?
       # GPO reminder emails include an "I did not receive my letter!" link that results in
       # slightly different copy on this screen.
       def user_did_not_receive_letter?
-        !!params[:did_not_receive_letter]
+        !!params[:did_not_receive_letter].present?
```

Comment:
`present?` is already only true or false
```suggestion
        params[:did_not_receive_letter].present?
```

Reply:
Thank you, fixed!